tests/alerts/geomodel/test_alert.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime, timedelta
+import unittest
```

Comment:
Since we use pytest to run our tests, we don't actually need to import unittest or inherit from unittest.TestCase, pytest will automatically detect and run it correctly.